prov/rxm/src/rxm_ep.c

```diff
@@ -280,6 +283,7 @@ int rxm_ep_prepost_buf(struct rxm_ep *rxm_ep, struct fid_ep *msg_ep)
 			rxm_buf_release(&rxm_ep->rx_pool, (struct rxm_buf *)rx_buf);
 			return ret;
 		}
+		dlist_insert_tail(&rx_buf->entry, &rxm_ep->post_rx_list);
 	}
 	return 0;
 }
```

Comment:
👍 for the patch but we may need to handle the error case where tx buffers aren't returned back to the buf pool.

Reply:
Thanks. Yes, cleaning the send queue to find any unreleased tx buf should work. Please go ahead with the merge.